webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -175,6 +185,47 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     containerHeight - 2 * parseInt(getCssVar('--theme-stroke-width')) - (isPagedView ? 40 : 0);
   const [scrollPositionSetCount] = useState(observable(0));
 
+  useEffect(() => {
+    if (!isLoadingSettings && !isLoading && experiments.some(Loadable.isLoaded)) {
```

Comment:
I think this last check `experiments.some(Loadable.isLoaded)` might actually be redundant with `!isLoading`.

Reply:
✔️ and isLoadingSettings too, since we finish that before fetching experiments